crates/rolldown/src/dev/dev_engine.rs

```diff
@@ -62,7 +62,7 @@ impl<W: Watcher + Send + 'static> DevEngine<W> {
       return;
     }
 
-    if let Some(build_process_future) = self.build_driver.schedule_build(vec![]).await {
+    if let Ok(Some(build_process_future)) = self.build_driver.schedule_build(vec![]).await {
       build_process_future.await;
     } else {
```

Comment:
I guess we should handle error here as well. (it shouldn't happen though)

Reply:
I think the callback approach would work. Would you add a comment or call `.expect` or something so that we don't forget to handle the error here?